contributing.md

```diff
@@ -0,0 +1,106 @@
+# Enarx RFCs
+
+## Contributing
+
+### Do you need an RFC?
+
+Use an RFC to advocate substantial changes to the Aries ecosystem, where
+those changes need to be understood by developers who *use* Aries. Minor
+changes are not RFC-worthy, and changes that are internal in nature,
+invisible to those consuming Aries, should be documented elsewhere.
```

Comment:
~~Aries~~ Enarx

Reply:
Yes, thanks, @lkatalin pointed out the same thing. I was more tired than i realised when i pushed this :)